Dockerfile

```diff
@@ -19,6 +20,9 @@ ARG build_type=Release
 ARG no_threads=1
 # Specify CMake arguments for Storm
 ARG cmake_args="-DSTORM_PORTABLE=ON"
+# Can be ON/OFF
+ARG gurobi_support="ON"
```

Comment:
I see the point of providing some convenience flags, but could we not use `cmake_args` instead? Then we have more flexibility to provide any CMake flags. We are also already using `cmake_args` to define the configurations in the CI.
The other option would be to add more on/off switches for all relevant configurations (spot, MathSAT, Cln, ...)

Reply:
I dislike the default then, because you need to carefully mimic what is already in the default. But maybe we can add the arguments hardcoded and one may overwrite them (as far as i understand, cmake takes the rightmost definition as the actual one). What do you think?

Reply:
We only need the PORTABLE flag as default argument to ensure platform independence. Should we simply hard-code this cmake flag and leave `cmake_args` empty? Then users can always provide their own flags.

Reply:
but how do we then set that the default is to include spot, soplex, etc in this build?

Reply:
Ah okay, you want to have the dependencies enabled by default. Then I agree that it is better to have separate flags for it.

Reply:
After this merge, I will create a new PR for some additional configurations: Spot, MathSat etc. I think it is better to have the changes to the Dockerfile in a separate PR because I should also adapt the deployed CI image then.